src/ray/common/ray_config_def.h

```diff
@@ -22,9 +22,13 @@
 RAY_CONFIG(uint64_t, debug_dump_period_milliseconds, 10000)
 
 /// Whether to enable Ray event stats collection.
-/// TODO(ekl) this seems to segfault Java unit tests when on by default?
 RAY_CONFIG(bool, event_stats, true)
 
+/// Whether to enbale Ray event stats metrics export.
+/// Note that enabling this adds high overhead to
+/// Ray metrics agent.
+RAY_CONFIG(bool, event_stats_metrics, false)
```

Comment:
this is an ambiguous naming though. What does "event stats" here mean? Could we doc more on what metrics it controlled?

Reply:
Actually, if you look at line 25, we already use this term (and this is internal usage only. Most of people don't know how to interpret it anyway).

Reply:
> Most of people don't know how to interpret it anyway)

haha, yeah, i am exactly one of those people. I did a grep on the relevant stuff - i guess you are right that it's used pretty universally. I guess it's just that it wasn't documented clear enough what it does in general.